lib/espejote.libsonnet

```diff
@@ -5,9 +5,10 @@
  */
 
 local helper = import 'component/helper.libsonnet';
-local roles = import 'component/roles.libsonnet';
+local roles = import 'component/roles_reading.libsonnet';
```

Comment:
Including files from `component/` isn't supported for component libraries. Relative imports are resolved when compiling each component with a set of directories to search based on that component. Therefore, `component/roles_reading.libsonnet` will not be found when compiling any component which uses this library (other than `espejote` where the search paths include `dependencies/espejote`).
 
Additionally, breaks with our current approach to include a component lib in other components' golden tests, since we only download the `lib/espejote.libsonnet` file.

Reply:
I see 🤔 that means I need to inclide the whole thing in `lib/espejote.libsonnet`.
hmm... including the reference to the operator namespace...

Reply:
we have precedents for reading `inv.parameters.<component>` in the library, so getting the namespace isn't really an issue

Reply:
You can segment it a bit like here: https://github.com/vshn/espejote/blob/main/controllers/lib/espejote.libsonnet

> including the reference to the operator namespace

That should be fine IMO as long as it is documented that it is accessed.

Reply:
Owee I need to export a lot of stuff into the lib directly 😬 
Can't we have a ManagedResource that creates reading RBAC for the other ManagedResources 😁

Reply:
It got a bit ugly